invokeai/app/services/image_storage.py

```diff
@@ -207,24 +207,26 @@ def save(
         self.__set_cache(image_path, image)
         self.__set_cache(thumbnail_path, thumbnail_image)
 
-        return (image_path, thumbnail_path, int(os.path.getctime(image_path)))
+        return (image_name, thumbnail_name, int(os.path.getctime(image_path)))
```

Comment:
I feel like I somehow missed the creation of this method. Please return an actual class. It's impossible to tell what each tuple member is supposed to represent otherwise (from the caller's end).